dotCMS/src/main/java/com/dotcms/rest/BundleResource.java

```diff
@@ -218,4 +229,250 @@ public Response deleteEnvironmentPushHistory(@Context HttpServletRequest request
         return responseResource.response( "true" );
 	}
 
+    /**
+     * Deletes all bundles by identifier
+     * Note: The process could be heavy, so it is handle by async response
+     * @param request
+     * @param response
+     * @param asyncResponse response is async
+     * @param deleteBundlesByIdentifierForm
+     */
+	@DELETE
+    @Path("/ids")
+    @Produces("application/json")
+    public void deleteBundlesByIdentifiers(@Context   final HttpServletRequest request,
+                                           @Context   final HttpServletResponse response,
+                                           @Suspended final AsyncResponse asyncResponse,
+                                           final DeleteBundlesByIdentifierForm  deleteBundlesByIdentifierForm) {
+
+        final InitDataObject initData = new WebResource.InitBuilder(webResource)
+                .requiredBackendUser(true)
+                .requiredFrontendUser(false)
+                .requestAndResponse(request, response)
+                .rejectWhenNoUser(true)
+                .init();
+
+        Logger.info(this, "Deleting the bundles: " + deleteBundlesByIdentifierForm.getIdentifiers()
+                + " by the user: " + initData.getUser().getUserId());
+
+        final DotSubmitter dotSubmitter = DotConcurrentFactory
+                .getInstance().getSubmitter(BUNDLE_THREAD_POOL_SUBMITTER_NAME);
+        dotSubmitter.execute(() -> {
+
+            Response restResponse = null;
+
+            try {
+
+                this.deleteBundleByIdentifier(deleteBundlesByIdentifierForm, initData);
+                restResponse = Response.ok(new ResponseEntityView(
+                        CollectionsUtils.map("bundlesDeleted", deleteBundlesByIdentifierForm.getIdentifiers()))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "bundlesDeleted" appears 5 times in this file; the first occurrence is on line 269](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)